src/ServiceFabric/ServiceFabric/Template/Windows/parameter.json

```diff
@@ -35,16 +35,16 @@
     "clusterProtectionLevel": {
       "value": "EncryptAndSign"
     },
-    "certificateStoreValue": {
+    "certificateStorevalue": {
```

Comment:
Value should stay capitalized in these three since it is capitalized in the template

Reply:
updated